src/System.Security.Cryptography.Encoding/src/Internal/Cryptography/OidLookup.OSX.cs

```diff
@@ -0,0 +1,30 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Security.Cryptography;
+
+namespace Internal.Cryptography
+{
+    internal static partial class OidLookup
+    {
+        private static bool ShouldUseCache(OidGroup oidGroup)
+        {
+            return true;
+        }
+
+        private static string NativeOidToFriendlyName(string oid, OidGroup oidGroup, bool fallBackToAllGroups)
+        {
+            return null;
```

Comment:
From the commit description:

> Apple's Security framework doesn't seem to expose any OID resolution or
ASN.1/DER pretty-printing, so the "make a native call here" functions just
always report that they have failed.

If it turns out that those functions do exist, we can always plumb them in later.